zephyr/src/dut_main.c

```diff
@@ -0,0 +1,249 @@
+/* Copyright (c) 2020 Wi-Fi Alliance                                                */
```

Comment:
Instead of copying OSS code, can we leverage those functions here and keep this file Zephyr specific (with nRF license)?